dev-libs/pcl/pcl-1.12.ebuild

```diff
@@ -0,0 +1,13 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+DESCRIPTION="Portable Coroutine Library"
+HOMEPAGE="http://xmailserver.org/libpcl.html"
+SRC_URI="http://xmailserver.org/pcl-${PV}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
```

Comment:
Suggest setting subslot to match SOVERSION of the library, i.e. `libpcl.so.1` -> `SLOT="0/1"`.